core/frontend/src/RealityDataConnection.ts

```diff
@@ -0,0 +1,118 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { GuidString } from "@bentley/bentleyjs-core";
+import { RealityDataSourceKey, RealityDataSourceProps } from "@bentley/imodeljs-common";
+import { RealityData, RealityDataClient } from "@bentley/reality-data-client";
+import { AuthorizedFrontendRequestContext } from "./FrontendRequestContext";
+import { RealityDataSource, realityDataSourceKeyToString } from "./RealityDataSource";
+
+/**
+ * This interface provide methods used to access a reality data from ContextShare
+ * @alpha
+ */
+export interface IRealityDataConnection {
+  getRealityData(): RealityData | undefined;
```

Comment:
Replace no-arg "get" methods with `readonly` properties.

Reply:
Why is `getSource()` still a function instead of a property?

Reply:
Push a new fix that changes RealityDataConnection.getSource() into a property instead of a function.